src/Microsoft.AspNet.Mvc.Razor/RazorViewEngineOptions.cs

```diff
@@ -65,5 +68,25 @@ public string Configuration
                 _configuration = value;
             }
         }
+
+        /// <summary>
+        /// Gets or sets the callback that is used to customize Razor compilation
+        /// to change compilation settings you can update <see cref="RoslynCompilationContext.Compilation"/> property
+        /// </summary>
+        /// <remarks>
+        /// At startup, this is initialized to empty lambda <code>(c => {})</code>.
```

Comment:
No need for the remarks. You cover this in the summary by stating it's used to `customize`. `Callback` also indicates that it doesn't do anything.